### DISCUSSION TITLE
Background

### DISCUSSION THREAD

**Discussion by JayOr209** [pege22/pege_source_main#46] [Polls]
Background

**Comment by LUIS9799** [comment]
https://strawpoll.com/polls/XOgOJKRprn3/results

**Comment by LUIS9799** [comment]
![image](https://user-images.githubusercontent.com/68512719/196573426-caad33b6-a84f-4919-a828-6e3eb88b4307.png)

**Comment by LUIS9799** [comment]
![image](https://user-images.githubusercontent.com/68512719/196573480-77bead43-3430-42ec-b52e-4fe2e588608d.png)
